draft-dekater-scion-controlplane.md

```diff
@@ -1403,7 +1403,7 @@ The control service of a non-core AS has to register the newly created down-segm
    message SegmentsRegistrationResponse {}
 ~~~~
 
-- `SegmentType`: Specifies the type of the path segment that must be registered. Currently, only the following type is used:
+- `SegmentType`: Specifies the type of the path segment that MUST be registered. Currently, only the following type is used:
```

Comment:
I feel that registering the segment is not an absolute requirement to the spec, I'd avoid RFC2119 wording: 
```suggestion
- `SegmentType`: Specifies the type of the path segment to be registered. Currently, only the following type is used:
```